pkg/util/sdnotify.go

```diff
@@ -0,0 +1,32 @@
+package util
+
+import (
+	"errors"
+	"net"
+	"os"
+)
+
+var NoNotifySocket = errors.New("NOTIFY_SOCKET environment variable empty or unset.")
+
+func SdNotify(state string) error {
+	notifySocket := os.Getenv("NOTIFY_SOCKET")
+
+	if notifySocket == "" {
+		return NoNotifySocket
+	}
+
+	socketAddr := &net.UnixAddr{
+		Name: notifySocket,
+		Net:  "unixgram",
+	}
+
+	conn, err := net.DialUnix(socketAddr.Net, nil, socketAddr)
```

Comment:
You are not closing this connection. Is that on purpose or did you for forget it? :)

Reply:
I forgot about it. I'll fix it :-)

Reply:
Ok, fixed.